.gitattributes

```diff
@@ -1 +1 @@
-pynvml/_version.py export-subst
+pynvml_utils/_version.py export-subst
```

Comment:
This file can be deleted. It was for versioneer. Plus we already deleted the `_version.py` file it references